odoo80/Dockerfile

```diff
@@ -25,12 +23,10 @@ RUN apt-get update -q && apt-get upgrade -q \
     postgresql-common \
     libpq-dev \
     libldap2-dev \
-    libsasl2-dev \
     libxml2-dev \
     libxslt1-dev  \
     xsltproc \
     xmlstarlet \
-    openssl \
     python-libxml2 \
     phantomjs \
     bash-completion
```

Comment:
Should we add a rm here like as https://github.com/nginxinc/docker-nginx/blob/aecb18605af6a33c1968c51df44e01e5f5a821e6/Dockerfile#L11 ?